.github/workflows/pull-request-python.yaml

```diff
@@ -11,23 +11,23 @@ on:
 
 jobs:
 
-  # Build is a single job to lint and test
   build:
     name: Build
     runs-on: ubuntu-latest
+    env:
+      IMAGE: ${{ github.repository }}
     steps:
     - name: Checkout
       uses: actions/checkout@v3
 
-    - name: Set up Python
-      uses: gramLabs/.github/.github/actions/setup-python@main
-      with:
-        python-version: '${{ inputs.python-version}}'
-
     - name: Lint with pylint
+      with:
+        image: '${{ github.repository	}}'
       run: |
-        sh lint.sh
+        docker build -t ${{ env.IMAGE }} --build-arg TARGET=lint .
```

Comment:
FWIW, there is no point in adding something to environment if you are just going to reference it from the context, i.e. this should be `-t "$IMAGE"` (which references the environment variable from the environment) or `-t "${{ github.repository }}"` (note that the later is invalid and there is no "to-lower-case" context expression to fix it, the former is "fixable" because you can use other tools from the shell to make it lower case).

Reply:
👍